### ISSUE TITLE
Seems like `optimizer.step()` has been overridden after learning rate scheduler initialization.

### ISSUE THREAD

**Issue by legolego** [ThilinaRajapakse/simpletransformers#107]
/usr/local/lib/python3.6/dist-packages/torch/optim/lr_scheduler.py:73: UserWarning: Seems like `optimizer.step()` has been overridden after learning rate scheduler initialization. Please, make sure to call `optimizer.step()` before `lr_scheduler.step()`. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)

I'm not sure of this is something in simpletransformers or not, but I'm running a notebook in paperspace and this warning comes up. I'm not sure if it's really a problem though, as it seems to be working.

**Comment by ThilinaRajapakse** [comment]
I'm not sure where this is coming from either as `optimizer.step()` is _not_ called before `lr_scheduler.step()`.

But it doesn't seem to be messing anything up so I think it's safe to let it be for now.
